fetch/src/main/scala/fetch.scala

```diff
@@ -657,16 +658,18 @@ object `package` {
       T: Clock[F]
   ): F[List[Request]] =
     for {
+      startTime   <- T.monotonic.map(_.toMillis)
```

Comment:
Would it be possible to pass the flag here to avoid unnecessary calls to `Clock`? It will be minimal, but in a library that aims to be performant, every ms counts 😄

Reply:
To be honest, I'm not sure. At first, I thought that for cached requests, the parameters "start" and "end" could be initialized with a constant, but it won't work because they are used to determine the first and last requests.